easy_phi/hwal.py

```diff
@@ -52,7 +66,6 @@ def is_instance(device):
         """
         return False
 
-    @lock
```

Comment:
@lock now applied automatically by metaclass (see line 35)
Metaclass was introduced to get rid of mandatory decorators in subclasses, I assume sooner or later somebody will forget to add it to a new class